Space and cosmos

zdrale opened the pull request:
For this project, we used HTML and CSS, this page consists of the following components:
-Header that is fixed on top.
-Content section with grid layout, aside section, and recommendation section laid out using CSS flex
-Bottom navigation which we laid out using float
-We also embedded images and videos

zdrale committed:
Initial commit

Added project files Index and styles

zdrale committed:
Add a Header

-Added structure of the header.
-Added image for main logo.
-Added CSS reset.

bini-i committed:
Add Style

- add style to header elements, buttons (font-size, colors and 
alignments)

zdrale committed:
Started second section of the main page

-Added image for banner
-Added new section
-Styled image and aligned elements

bini-i committed:
Add Article

- added semantic structure of article
- styled and aligned main, article and aside section.

zdrale committed:
Add inflation region

-Added image for inflation section
-Added subscribe link

zdrale committed:
Add aside element

-Started working on aside element
-Added 8 rows to the grid

bini-i committed:
Add Theory of inflation section

- add semantic structure
- style alignment

zdrale committed:
Add aside section

-Added images
-Styled aside section

bini-i committed:
Add advertisement sections and add padding to aside

bini-i committed:
Update figure and image 

-update alignemt and font property of figcaptions

zdrale committed:
Updated header to stay fixed on top

Added Button and social links

bini-i committed:
Add Recommendation section

- add semantic structure of recommendation section
- add styling, alignment

zdrale committed:
Added Navigation on bottom of the page

bini-i committed:
Add footer

-add footer semantic structure
-add alignment and styling

bini-i committed:
Add README.md file

zdrale committed:
Update README.md

bini-i committed:
Add linter configuration and Update css comments

bini-i committed:
update reset css

bini-i committed:
Merge branch 'Space-and-cosmos' of github.com:zdrale/NyTimesClone into Space-and-cosmos

bini-i committed:
Update for linters requriment

bini-i committed:
update for linters' requirement

bini-i committed:
update for linter requriement

jubaan reviewed (changes requested):
# Status: Changes Required :recycle:

## :wave: :wave: :wave: Hello team!!! Awesome work!! :clap: :clap: :clap:  it looks close enough to the original website! :100: 

Things to notice:
:100: Use of semantic HTML
:100: Great looking clone website! very similar to the original!
:100: Great use of ```grid```, ```flex``` and ```float```
:100: Passes al linter checks

The page looks amazing guys, Congrats! still there are some minor issues to be fixed before moving forward. Please work on them.

## Required Changes
### [Mandatory]

- [x] NYT logo in navbar should be a link
![image](https://user-images.githubusercontent.com/25774518/93902370-159f4780-fcbd-11ea-8314-b90759503886.png)

- [x] Content is not centered
![image](https://user-images.githubusercontent.com/25774518/93902583-55fec580-fcbd-11ea-8d76-df34370ab2d4.png)
![image](https://user-images.githubusercontent.com/25774518/93902663-7464c100-fcbd-11ea-8c8f-3d27d40e3220.png)
![image](https://user-images.githubusercontent.com/25774518/93902729-85adcd80-fcbd-11ea-8ac3-0a7804367014.png)

### [OPTIONAL]

- [ ] see inline comments

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

jubaan commented on index.html:
```diff
@@ -0,0 +1,607 @@
+    <main>
+      <article>
+        <div class="about">
+          <div class="author-detail">
+
+            <h2 class="author">By <a href="http://www.nytimes.com/by/dennis-overbye">Dennis Overbye</a></h2>
+
+            <p class="date">March 17, 2014</p>
```
- [ ] Try to keep the code tidy and compact by separating the code only for bigger chunks like sections. Remove the white space.

jubaan commented on index.html:
```diff
@@ -0,0 +1,607 @@
+            </div>
+            <p>FLUCTUATION Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves
+              predicted
+              by Einstein.</p>
+
+          </div>
```
Blank line not needed... review the code and remove where necessary.

jubaan commented on index.html:
```diff
@@ -0,0 +1,607 @@
+
+          </div>
+        </div>
+      </div>
+
+
+
+      <article class="content">
+        <p>Dr. Kovac said the chance that the results were a fluke was only one in 10 million.</p>
```
- [ ] Maintain the formatting style through the code. Remove extra lines

jubaan commented on index.html:
```diff
@@ -0,0 +1,607 @@
+      <div class="adv2">ADVERTISEMENT</div>
+
+      <article class="content">
+        <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the
+          cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.</p>
+
+        <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+
+        <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+
+        <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
```
- [ ] Maintain long lines of code between 80 or fewer columns. Breaking long lines into new lines improves readability.

jubaan commented on css/style.css:
```diff
@@ -0,0 +1,556 @@
+  font-size: 15px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
```
- [ ] Prefer HTML class over general selectors. Improves readability and specificity. Example:
```
.nav__link {
    ... some properties...
}
```

jubaan commented on css/style.css:
```diff
@@ -0,0 +1,556 @@
+  font-weight: bold;
+}
+
+.adv1,
+.adv2 {
```
- [x] Prefer meaningful names rather than abbreviations. It gives the programmer fewer things to think about. Example:
```
.advertisement {
    ... some properties...
}
```

jubaan reviewed (changes requested):
[OPTIONAL]

bini-i committed:
update for reviews made

- center alignement fixed
- logo given a link
- removed extra blank lines
- updated class names to meaningful names

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @bini-i AND @zdrale 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/93986793-2c6a9a80-fda0-11ea-8632-b830decb19aa.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/)




- [ ] The `Subscribe` & `Newsletter` sections still need some work to do, you can notice the height of the two sections is not matching, also as requested by previous TSE, the newsweek logo must match the one from the original website


### Footer
- [ ] SVGs are missing in footer

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/zdrale/NyTimesClone/Space-and-cosmos/index.html)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/93986989-705d9f80-fda0-11ea-896b-88489c27ad72.png)|![clone](https://user-images.githubusercontent.com/29689273/93987037-80757f00-fda0-11ea-9c7d-1d9ebdda6e99.png)|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

bini-i committed:
update for review

- added svg in recommendations section 
- match height of subscribe and newsletter sections height

arslanbisharat reviewed (approved):
### Congratulations APPROVED :clap:
Hello, I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS 💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat
